examples/search_examples/public/components/app.tsx

```diff
@@ -162,7 +164,7 @@ export const SearchExamplesApp = ({
               text: mountReactNode(message),
             });
             searchSubscription$.unsubscribe();
-          } else if (response.isPartial && !response.isRunning) {
```

Comment:
Do we know this is an actual documented error case? Is there a reliable way to reproduce this error?

Reply:
I pinged the security team, for them to consider reducing code duplication.

And this is definitely a documented error case (this is what delayed the release of 7.8 or 7.7 I remember)